proto/reserve/vaults/proposal.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+package reserve.vaults;
+
+
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+import "amino/amino.proto";
+
+option go_package = "github.com/onomyprotocol/reserve/x/vaults/types";
+
+message ActiveCollateralProposal {
```

Comment:
We should add a param `oracleScriptId` to support query price of asset in oracle module
cc: [here](https://github.com/onomyprotocol/reserve/blob/e1c9abc07db8594bcaa4380cfca5f79457c72775/x/oracle/keeper/band_oracle.go#L245)